regolith/helpers/a_presentationhelper.py

```diff
@@ -0,0 +1,148 @@
+"""Helper for adding a presentation to the presentation collection.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "presentations"
+ALLOWED_TYPES = ["award", "keynote", "plenary", "invited", "contributed_oral",
+                 "poster", "colloquium", "seminar","other"]
+ALLOWED_STATI = ["accepted"]
+
+
+def subparser(subpi):
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("place", help="the place of the presentation, location if conference,"
+                                     "institution for seminars"
+                       )
+    subpi.add_argument("name", help="name of the presentation, meeting name if meeting,"
+                                    "department if seminar",
+                       )
+    subpi.add_argument("-t","--type", help=f"types, from {ALLOWED_TYPES}. Default is other",
+                       default="invited"
+                       )
+    subpi.add_argument("-d", "--begin_date",
```

Comment:
I think begin and end date should be required.  It doesn't make any sense to have them default to todays day.

Reply:
done